src/mame/igs/pgm.cpp

```diff
@@ -4118,6 +4118,35 @@ ROM_START( theglad101 )
 	ROM_LOAD( "igs_w04601b64m.u1",    0x400000, 0x800000, CRC(5f15ddb3) SHA1(c38dcef8e06802a84e42a7fc9fa505475fc3ac65) )
 ROM_END
 
+ROM_START( theglad104 )
+	ROM_REGION( 0x600000, "maincpu", 0 ) /* 68000 Code */
+	PGM_68K_BIOS
+	ROM_LOAD16_WORD_SWAP( "v100.u6",       0x100000, 0x080000, CRC(bcf3b172) SHA1(df7e2808c0341be0a59eefa852c857a3a919223e) ) // 02/25/03 17:42:51
+
+	ROM_REGION( 0x4000, "prot", 0 ) /* ARM protection ASIC - internal rom */
+	ROM_LOAD( "theglad_igs027a_execute_only_area", 0x0000, 0x00188, NO_DUMP )
+	ROM_LOAD( "theglad_igs027a_v100_overseas.bin", 0x0188, 0x3e78, CRC(02fe6f52) SHA1(0b0ddf4507856cfc5b7d4ef7e4c5375254c2a024) )
```

Comment:
Should this be marked bad if it’s taken from another set?

Reply:
Possibly, yeah. I based this line on the other `theglad` revisions, which all share this one ROM without marking it as a bad dump even though it wasn't dumped for most of them.

Reply:
Given Haze says it has region bytes and they're not changed, could you mark it bad dump?

Reply:
Done. Sorry for taking awhile getting back to you!